mediathread/assetmgr/models.py

```diff
@@ -339,3 +340,30 @@ def __unicode__(self):
 
     class Meta:
         ordering = ['title']
+
+
+class JuxtaposeAsset(models.Model):
+    added = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    author = models.ForeignKey(User)
+    course = models.ForeignKey(Course)
+    title = models.TextField()
+    spine = models.ForeignKey(SherdNote, blank=True, null=True)
+
+
+class JuxtaposeMediaItem(models.Model):
```

Comment:
Putting on my Mark hat here: I'm concerned by overloading the terms "Item", "range1" and "range2".

"Item" refers to something very specific in Mediathread, e.g. a whole video, image or audio track. Perhaps "Clip" instead?

"range1" and "range2" is how the code denotes the portion of the larger item or asset. Perhaps "cue1" and "cue2"?